reports/report.tex

```diff
@@ -35,7 +35,25 @@
 \maketitle
 
 \begin{abstract}
-  \colorbox{yellow}{TODO}
+
+We explore automated model-checking of the Ethereum specification, specifically
+the accountable safety of the 3SF protocol. We use~\tlap{} as our primary
+specification language and Apalache model checker with SMT solver Z3. As a
+first step, we manually translated the Python specification into~\tlap{}. This
```

Comment:
I really think the abstract should all be in present tense.

Reply:
Good point. Fixed that.